ClientApp/.env

```diff
@@ -0,0 +1,17 @@
+SKIP_PREFLIGHT_CHECK=true
+
+REACT_APP_AUTH_CLIENT_ID='ExitSurveyAdmin'
+
+# No trailing slash
+REACT_APP_APP_DOMAIN='http://localhost:3000'
+REACT_APP_API_DOMAIN='http://localhost:5050'
+
+# MUST have a trailing slash!
+REACT_APP_APP_PATH='/'
+
+REACT_APP_AUTH_RESPONSE_TYPE='code'
+REACT_APP_AUTH_SCOPE='openid profile resourceApi'
+REACT_APP_AUTH_URL='https://dev.oidc.gov.bc.ca/auth/realms/ytaqhqia'
```

Comment:
at the very least, the values here wouldn't be hard coded, in some cases because they might be sensitive and/or are better served as variables that can be set depending on which environment it's being deployed to.

Reply:
might even be better to consider adding `.env` to the `.gitignore` and add an example template and instructions in the readme.

Reply:
Tracking in #35.

Reply:
PR: https://github.com/bcgov/NewJobSurveyAdmin/pull/36